### ISSUE TITLE
Create new CrocodileJS project fails

### ISSUE THREAD

**Issue by aluengo** [crocodilejs/crocodile-node-mvc-framework#221]
Hi!

I'm new to Node but after what I've read Crocodile looks promising and I would like to test it. I've installed NVM, Node (v6.7.0) and NPM (3.10.3). Crocodile version is 1.0.2

I had problems when installing Crocodile >> "Cannot resolve module babel-runtime/regenerator" >> so I installed babel-runtime via "npm install -g babel-runtime"

After that I installed Crocodile with no problem >> "npm install -g crocodile"

Now, when I try to create a new project "crocodile chew myApp" I get an error when cloning the repo. This is what I get:

```
✗ An error occured while cloning the repository from GitHub, please try again or file an issue with the below output:

Error: Command failed: git describe --tags `git rev-list --tags --max-count=1`
usage: git rev-list [OPTION] <commit-id>... [ -- paths... ]
  limiting output:
    --max-count=<n>
    --max-age=<epoch>
    --min-age=<epoch>
    --sparse
    --no-merges
    --min-parents=<n>
    --no-min-parents
    --max-parents=<n>
    --no-max-parents
    --remove-empty
    --all
    --branches
    --tags
    --remotes
    --stdin
    --quiet
  ordering output:
    --topo-order
    --date-order
    --reverse
  formatting output:
    --parents
    --children
    --objects | --objects-edge
    --unpacked
    --header | --pretty
    --abbrev=<n> | --no-abbrev
    --abbrev-commit
    --left-right
    --count
  special purpose:
    --bisect
    --bisect-vars
    --bisect-all
fatal: No names found, cannot describe anything.
 `git describe --tags `git rev-list --tags --max-count=1`` (exited with error code 128)
    at ChildProcess.exithandler (child_process.js:206:12)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:877:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)
```

Am I doing anything wrong?

Thanks a lot!

**Comment by niftylettuce** [comment]
@aluengo Hey!  You're getting this error because I haven't yet released the project!  V1.0.0 should be out this week.  Join us in Slack and you'll be the first to know when it's available to install!  👍 

<http://slack.crocodilejs.com/>

**Comment by aluengo** [comment]
Thanks very much @niftylettuce I'll wait for it!!
